crates/proof-of-sql/src/sql/parse/query_expr_tests.rs

```diff
@@ -1683,7 +1721,7 @@ fn varchar_column_is_not_allowed_within_numeric_aggregations() {
             if expression == "cannot use expression of type 'varchar' with numeric aggregation function 'max'"
     ));
 
-    let min_query = "select min(s) from sxt.t";
+    let min_query = "select min(name) from sxt.employees";
```

Comment:
Same.

Reply:
No, the change I made is in relevance to test `varchar_column_is_not_allowed_within_numeric_aggregations` because the test is checking for the error `cannot use expression of type 'varchar' with numeric aggregation function 'min'` to be logged.

Reply:
Same here, this seems fine to me.